app/src/main/java/com/example/lamnn/todo_mvp/data/repository/AsyncTaskTasksContent.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lamnn.todo_mvp.data.repository;
+
+import android.os.AsyncTask;
+
+import com.example.lamnn.todo_mvp.data.model.Task;
+import com.example.lamnn.todo_mvp.data.source.Callback;
+
+import org.json.JSONException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AsyncTaskTasksContent extends AsyncTask<String, Void, List<Task>> {
```

Comment:
nên để hậu tố là asyntask thay vì tiền tố 
```suggestion
public class TaskLoaderAsyncTask extends AsyncTask<String, Void, List<Task>> {
```